src/browzine-summon-adapter.js

```diff
@@ -98,56 +124,103 @@ angular.module("summonApp.directives")
-                     "{{assetClass}}: <a class='browzine-web-link' href='{{browzineWebLink}}' target='_blank' style='text-decoration: underline; color: #333;'>Browse Now</a> " +
-                     "<img class='browzine-book-icon' src='{{bookIcon}}'/>" +
+                     "{wording}: <a class='browzine-web-link' href='{browzineWebLink}' target='_blank' style='text-decoration: underline; color: #333;'>{browzineWebLinkText}</a> " +
+                     "<img class='browzine-book-icon' src='{bookIcon}'/>" +
                    "</div>";
 
-    template = template.replace(/{{assetClass}}/g, assetClass);
-    template = template.replace(/{{browzineWebLink}}/g, browzineWebLink);
-    template = template.replace(/{{bookIcon}}/g, bookIcon);
+    template = template.replace(/{wording}/g, wording);
+    template = template.replace(/{browzineWebLink}/g, browzineWebLink);
+    template = template.replace(/{browzineWebLinkText}/g, browzineWebLinkText);
+    template = template.replace(/{bookIcon}/g, bookIcon);
 
     return template;
   };
 
-  return {
-    link: function(scope, element, attributes) {
-      if(!shouldEnhance(scope)) {
-        return;
-      }
+  function getScope(documentSummary) {
+    return angular.element(documentSummary).scope();
+  };
 
-      var endpoint = getEndpoint(scope);
+  function resultsWithBrowZine(documentSummary) {
+    var scope = getScope(documentSummary);
 
-      http.get(sce.trustAsResourceUrl(endpoint)).then(function(response) {
-        var data = getData(response);
-        var browzineWebLink = getBrowZineWebLink(data);
-        var coverImageUrl = getCoverImageUrl(data, response);
+    if(!shouldEnhance(scope)) {
+      return;
+    }
 
-        if(browzineWebLink) {
-          var template = buildTemplate(data, browzineWebLink, bookIcon);
-          element.find(".docFooter .row:first").append(template);
-        }
+    var endpoint = getEndpoint(scope);
 
-        if(coverImageUrl) {
-          element.find(".coverImage img").attr("src", coverImageUrl);
-        }
-      });
-    }
+    $.getJSON(endpoint, function(response) {
+      var data = getData(response);
+      var browzineWebLink = getBrowZineWebLink(data);
+      var coverImageUrl = getCoverImageUrl(data, response);
+
+      if(browzineWebLink) {
+        var template = buildTemplate(data, browzineWebLink);
+        $(documentSummary).find(".docFooter .row:eq(0)").append(template);
+      }
+
+      if(coverImageUrl) {
+        $(documentSummary).find(".coverImage img").attr("src", coverImageUrl);
+      }
+    });
+  };
+
+  return {
+    resultsWithBrowZine: resultsWithBrowZine,
+    getScope: getScope,
+    shouldEnhance: shouldEnhance,
+    getEndpoint: getEndpoint,
+    getData: getData,
+    getBrowZineWebLink: getBrowZineWebLink,
+    getCoverImageUrl: getCoverImageUrl,
+    buildTemplate: buildTemplate,
+  };
+}());
+
+$(function() {
+  if(typeof browzine === "undefined" || browzine === null) {
```

Comment:
Is this meaningfully different from `if(!browzine) {`?

Reply:
Awww, good catch! Looks like `!browzine` checks [more cases](https://stackoverflow.com/a/5941649):
```
false
null
undefined
""
0
NaN
```

Whereas the above only checks or `undefined` and `null`. Thanks!